src/main/java/arpa/home/nustudy/command/HelpCommand.java

```diff
@@ -37,7 +37,8 @@ public void execute(CourseManager courses, SessionManager sessions) {
                  Filter by course                   filter <course>                     filter MA1511
                  Filter by date                     filter <date>                       filter 23/10/2025
                  Filter code and date               filter <course> <date>              filter MA1511 23/10/2025
-                 Reset                              reset <course> or reset all         reset CS2113
+                 Reset a Course                     reset <course>                      reset CS2113
+                 Reset all Courses                  reset all                           reset all
```

Comment:
Random capitalisation of "Course"


```suggestion
                 Reset a course                     reset <course>                      reset CS2113
                 Reset all courses                  reset all                           reset all
```